projects/adf-office-services-ext/src/lib/evaluators.spec.ts

```diff
@@ -303,5 +320,12 @@ describe('evaluators', () => {
 
       expect(canOpenWithOffice(context)).toBeTruthy();
     });
+
+    it('should return [false] when `aosPlugin` property is false', () => {
```

Comment:
can we add a new test that checks only the asoPlugin in case of true?

Reply:
It would be complicated. Since I added the validation of the AOS plugin at the beggining of the `canOpenWithOffice` method, I can't really return true unless I have all of the other validations to be true as well. And we are testing that already in the 'should return [true] if all checks succeed' test.